### DISCUSSION TITLE
Horizontal scrolling struggle

### DISCUSSION THREAD

**Discussion by GraceGreen02** [Budibase/budibase#14261] [Help]
Please help me. I am new with Budibase. I have no idea how to make a page that can scroll both horizontally and vertically. This page contains a lot of cards.

**Comment by ConorWebb96** [comment]
I'd recommend having a look into the [embed component.](https://docs.budibase.com/docs/embed) Plus custom CSS that you can write that will allow for overflowing vertically and horizontally.
